webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -673,7 +673,7 @@ protected function validateAndUpdateContest(string $entityType, ?string $eventId
 
         // The timezones are given in ISO 8601 and we only support names.
         // This is why we will use the platform default timezone and just verify it matches.
-        $startTime = $data['start_time'] === null ? null : new DateTime($data['start_time']);
```

Comment:
What problem does this solve?

Reply:
If start time is omitted from the json instead of being null

Reply:
Do we know of a tool which does that?

Reply:
We do. The cds removes null fields